build/__python-versions.sh

```diff
@@ -1,6 +1,9 @@
 # This file was auto-generated from 'build-constants.yaml'. Changes may be overridden.
 
+PYTHON_BUILD_BASE_TAG='20190610.2'
+PYTHON_RUNTIME_BASE_TAG='20190402.1'
 PIP_VERSION='19.0.3'
 PYTHON27_VERSION='2.7.16'
 PYTHON36_VERSION='3.6.8'
 PYTHON37_VERSION='3.7.3'
+PYTHON38_VERSION='3.8.0b1'
```

Comment:
are you planning to add some E2E tests?

Reply:
added one for now. will add more in a separate PR.